test/resilient/circuit_breaker_test.rb

```diff
@@ -5,13 +5,29 @@
 module Resilient
   class CircuitBreakerTest < Resilient::Test
     def setup
-      @object = CircuitBreaker.new(key: Resilient::Key.new("test"))
+      super
+      @object = CircuitBreaker.new(key: Resilient::Key.new("object"))
```

Comment:
Had to change this key to not conflict with other tests that don't use this setup object.